### ISSUE TITLE
Support for m4b Files

### ISSUE THREAD

**Issue by enxio** [geigi/cozy#333]
Hi thanks very much for this!

I have a few audiobooks in m4b format which the importer apparently does not recognize. I did apply the attached patch and now, after re-importing they show up and play just fine.

Maybe this could be considered?

Thanks!

```diff --git a/cozy/media/media_detector.py b/cozy/media/media_detector.py
index 780edd4..8da9e17 100644
--- a/cozy/media/media_detector.py
+++ b/cozy/media/media_detector.py
@@ -32,7 +32,7 @@ class MediaDetector(EventSender):
         self.discoverer: GstPbutils.Discoverer = GstPbutils.Discoverer()
 
     def get_media_data(self) -> MediaFile:
-        if not self.uri.endswith(('.mp3', '.ogg', '.flac', '.m4a', '.wav', '.opus')):
+        if not self.uri.endswith(('.mp3', '.ogg', '.flac', '.m4a', '.m4b', '.wav', '.opus')):
             raise NotAnAudioFile
 
         try:
```

**Comment by geigi** [comment]
Hey, thanks for creating the issue! Yes this is now possible, but I haven't thought about including this yet :) Chapters within the m4b are not detected for now though. This will be the next big feature for Cozy.

One thing that came to my mind: When Cozy does support chapters for m4b in the future, the listening progress of the book must transition correctly to the chapter after updating Cozy. But otherwise this might just work for now without chapter support :)

I'll release a bugfix version in the next few days and I'll include this also :)

**Comment by geigi** [comment]
Implemented in 933df4c75465e23a1b4eafcbee3d47f548beff6c
